gossip/src/cluster_info.rs

```diff
@@ -1358,12 +1358,16 @@ impl ClusterInfo {
     }
 
     fn is_spy_node(node: &LegacyContactInfo, socket_addr_space: &SocketAddrSpace) -> bool {
-        ![node.tpu(), node.gossip(), node.tvu()]
-            .into_iter()
-            .all(|addr| {
-                addr.map(|addr| socket_addr_space.check(&addr))
-                    .unwrap_or_default()
-            })
+        ![
+            node.tpu(contact_info::Protocol::QUIC),
```

Comment:
It looks like this one switched from UDP (`node.tpu()`) to QUIC (`node.tpu(contact_info::Protocol::QUIC`) - was that intentional ?

Reply:
Reverted back to UDP.
Will update those in a separate follow up commit.